pytorch_lightning/trainer/callback_config.py

```diff
@@ -109,20 +107,22 @@ def configure_early_stopping(self, early_stop_callback):
             self.early_stop_callback = early_stop_callback
             self.enable_early_stop = True
 
-    def configure_progress_bar(self):
+    def configure_progress_bar(self, refresh_rate=1, process_position=0):
         progress_bars = [c for c in self.callbacks if isinstance(c, ProgressBarBase)]
         if len(progress_bars) > 1:
             raise MisconfigurationException(
```

Comment:
I ques we can allow multiple in future, just check that each is different, meaning another monitor event, frequency, etc.

Reply:
yes agree that would be good. the only reason why we currently have a limit of 1 is because Trainer needs to be able to disable the progress bar temporarily, for example in LRFinder.